extensions/resource-deployment/src/test/index.ts

```diff
@@ -9,7 +9,6 @@ const testRunner = require('vscodetestcover');
 const suite = 'resource-deployment Extension Tests';
 
 const mochaOptions: any = {
-	ui: 'tdd',
```

Comment:
Let's keep this specific and put 'bdd' for the ones we want to use bdd.

Reply:
I wanted to use bdd for all of them.

Reply:
Yeah that's fine - I'm just saying specify `ui: 'bdd'` instead of leaving it as the default (even if that is already bdd)

Reply:
done